.github/workflows/fork_sync.yml

```diff
@@ -15,5 +15,5 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           owner: firebase
-          base: version-7.2.0-dev
-          head: compose-dev
\ No newline at end of file
+          base: compose-dev
+          head: version-7.2.0-dev
```

Comment:
I'm not sure if we should use `v7.2.0` or `master` 🤔 

Because v7.2.0 might get released before we finish the work here and then we'd have to change the head branch to the next release, I guess?

Reply:
Yeah, exactly. `v7.2.0` has some changes (the activity result apis) which we need, later on we can rebase `compose-dev` on `v8.0.0` to target that. I suspect that's going to be some work anyways so we're fine having to switch it here, just gotta remember :D